6. Developer Guides/09. Realtime-API/1. Method Calls/06. Get Subscriptions/README.md

```diff
@@ -10,3 +12,50 @@ TOOD: I have no returned data here. Need back up.
     "params": [ { "$date": 1480377601 } ]
 } 
 ```
+
+Response:
+
+```json
+{
+    "msg": "result",
+    "id": "42",
+    "result": [
+        ... // subcriptions
+    ]
+}
+```
+
+## Subscription Object
+
+The subscription object contains the information about the room and the user relation to it.
```

Comment:
@laggedHero I added some details for the subscription object, take a look and let me know if you have any questions.